src/audio/pcm_converter/pcm_converter_generic.c

```diff
@@ -841,6 +847,10 @@ const struct pcm_func_vc_map pcm_func_vc_map[] = {
 	{ SOF_IPC_FRAME_S32_LE, SOF_IPC_FRAME_S24_4LE, SOF_IPC_FRAME_S32_LE, SOF_IPC_FRAME_S16_LE,
 		ipc4_gtw_alh, ipc4_playback, pcm_convert_s24_to_s32 },
 #endif
+#if CONFIG_PCM_CONVERTER_FORMAT_U8 && CONFIG_PCM_CONVERTER_FORMAT_S16LE
+	{ SOF_IPC_FRAME_S16_LE, SOF_IPC_FRAME_U8, SOF_IPC_FRAME_S16_LE, SOF_IPC_FRAME_U8,
+		ipc4_gtw_all, ipc4_bidirection, audio_stream_copy },
```

Comment:
why S16LE and U8 convert still use audio_stream_copy? it should use new converter function, if new function in next patch, we need squash them into one.

Reply:
There is no conversion here. It is a simple copy of 8in16 frames to 8in16.